x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/defense_evasion_execution_lolbas_wuauclt.json

```diff
@@ -48,5 +48,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 5
+  "version": 7
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950